mtest/zerberus/comments/CMakeLists.txt

```diff
@@ -18,4 +18,8 @@ include_directories(
       ${SNDFILE_INCDIR}
       )
 
-target_link_libraries(tst_sfzcomments zerberus synthesizer audiofile ${SNDFILE_LIB} testutils)
+if (NOT MSVC)
+      target_link_libraries(tst_sfzcomments zerberus synthesizer audiofile ${SNDFILE_LIB} testutils)
+else (NOT MSVC)
+      target_link_libraries(tst_sfzcomments zerberus synthesizer audiofile sndfiledll testutils)
+endif (NOT MSVC)
```

Comment:
I would rewrite this (and the 6 others like it) as follows:
```
if (MSVC OR MINGW)
      target_link_libraries(tst_sfzcomments zerberus synthesizer audiofile sndfiledll testutils)
else (MSVC OR MINGW)
      target_link_libraries(tst_sfzcomments zerberus synthesizer audiofile ${SNDFILE_LIB} testutils)
endif (MSVC OR MINGW)
```
See https://github.com/musescore/MuseScore/blob/a5c92e2040dae24a09e4cd8bc2a50e61ba2cd579/mscore/CMakeLists.txt#L525-L529 for reference.

Reply:
Thank you. Done.